Configuration/Services.yaml

```diff
@@ -4,9 +4,17 @@ services:
     autoconfigure: true
     public: false
 
+  Causal\Oidc\:
+    resource: '../Classes/*'
+
   Causal\Oidc\EventListener\FrontendLoginEventListener:
     tags:
       - name: event.listener
         identifier: 'causal/oidc'
         method: 'modifyLoginFormView'
         event: TYPO3\CMS\FrontendLogin\Event\ModifyLoginFormViewEvent
+
+  Causal\Oidc\EventListener\ProcessRequestTokenListener:
+    tags:
+      - name: event.listener
+        identifier: 'causal/oidc-request-token'
```

Comment:
Which event are we listening to?

Reply:
It is this Event: BeforeRequestTokenProcessedEvent
We introduced this. See this commit:
https://github.com/undkonsorten/t3ext-oidc/commit/f2e4250f74929468b1369a30ae3423af36b799af

Reply:
This is needed for CSRF in typo3 12 for fe_login.
But this is breaking and might not work under typo3 11.